code/modules/power/apc.dm

```diff
@@ -64,39 +79,39 @@
 	var/obj/item/stock_parts/cell/cell
 	var/start_charge = 90				// initial cell charge %
 	var/cell_type = 2500	//Base cell has 2500 capacity. Enter the path of a different cell you want to use. cell determines charge rates, max capacity, ect. These can also be changed with other APC vars, but isn't recommended to minimize the risk of accidental usage of dirty editted APCs
-	var/opened = 0 //0=closed, 1=opened, 2=cover removed
-	var/shorted = 0
+	var/opened = APC_CLOSED
+	var/shorted = FALSE
 	var/lighting = 3
 	var/equipment = 3
 	var/environ = 3
-	var/operating = 1
-	var/charging = 0
+	var/operating = TRUE
+	var/charging = APC_NOT_CHARGING
```

Comment:
This field is used in multiple files. You can check this in vscode by right clicking on the field and selecting "Find all references"
![image](https://user-images.githubusercontent.com/15887760/178094186-ce70752d-f8cc-45d2-a126-034f54198813.png)

Reply:
I wish I'd known about checking references like that two weeks ago - I guess that's what I get for not using a proper IDE for years.